test/L2/paused/L2VestingWalletPaused.t.sol

```diff
@@ -72,6 +76,41 @@ contract L2VestingWalletPausedTest is Test {
             abi.encodeWithSelector(l2VestingWalletPausedImplementation.initializePaused.selector)
         );
         vm.stopPrank();
+
+        // Wrapping with L2VestingWalletPaused
+        l2VestingWalletPaused = L2VestingWalletPaused(payable(address(l2VestingWallet)));
+    }
+
+    function test_CustodianAddress() public view {
+        // Address of Security Council on L2
+        assertEq(l2VestingWalletPaused.custodianAddress(), 0x394Ae9d48eeca1C69a989B5A8C787081595c55A7);
+    }
+
+    function test_SendToCustodian_RevertWhenNotContractAdmin() public {
+        address nobody = vm.addr(1);
+
+        vm.startPrank(nobody);
+        vm.expectRevert(
+            abi.encodeWithSelector(
+                IAccessControl.AccessControlUnauthorizedAccount.selector, nobody, l2VestingWallet.CONTRACT_ADMIN_ROLE()
+            )
+        );
+        l2VestingWalletPaused.sendToCustodian(IERC20(address(mockToken)));
+        vm.stopPrank();
+    }
+
+    function test_SendToCustodian() public {
+        uint256 contractBalanceBefore = mockToken.balanceOf(address(l2VestingWalletPaused));
+        uint256 recoveryAddressBalanceBefore = mockToken.balanceOf(l2VestingWalletPaused.custodianAddress());
+
+        vm.startPrank(contractAdmin);
+        l2VestingWalletPaused.sendToCustodian(IERC20(address(mockToken)));
+        vm.stopPrank();
+
+        assertEq(
+            mockToken.balanceOf(l2VestingWalletPaused.custodianAddress()),
+            contractBalanceBefore + recoveryAddressBalanceBefore
+        );
```

Comment:
To nitpick, perhaps the balance of the contractAddress is zero can be asserted but its mostly redundant so your choice. Rest seems 👌 .

Reply:
Should be fine as tokens are sent to contract at setUp stage. 👍